CHANGES.rst

```diff
@@ -6,6 +6,12 @@
   - Protobuf with ``protoc`` [GH-1125]
   - systemd-analyze with ``systemd-analyze`` [GH-1135]
 
+- Improvements:
+
+  - Use ``flycheck-go-build-tags`` to specify go build tags for
+    ``go-test``, ``go-vet`` and ``go-errcheck`` checkers. Previously
+    this option was only used in the ``go-build`` checker.
```

Comment:
Please remove the second sentence, and shorten the first one to ````Use option ``flycheck-go-build-tags`` to  for  ``go-test``, ``go-vet`` and ``go-errcheck``as well````.  The changelog should be brief.